gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -263,4 +292,90 @@ public static ProjectListOption fields(ProjectField... fields) {
    * @throws ResourceManagerException upon failure
    */
   void undelete(String projectId);
+
+  /**
+   * Returns the IAM access control policy for the specified project. Returns null if the resource
```

Comment:
No, I think that this is good enough. It was more of a conceptual question. In other places, we throw exception if a container does not exist (e.g., trying to get a DNS record from a non-existent zone). If we consider a project to be a container for the policy, we should consider the same approach. If we consider policy to be an independent entity which may be associated to and deassociated from a project, then this is fine.

(If you discussed this before, feel free to ignore this comment.)